src/ast_tag.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "node_allocator.hpp"
```

Comment:
why would `ast_tag.hpp` need `node_allocator.hpp"?

Reply:
That was an oopsie